### ISSUE TITLE
when i run the command "python3 -m vit_jax.train --name ViT-B_16-cifar10_`date +%F_%H%M%S` --model ViT-B_16 --logdir /tmp/vit_logs --dataset cifar10",I encountered the error"Error executing an HTTP request: libcurl code 6 meaning 'Couldn't resolve host name', error details: Couldn't resolve host 'metadata'Error executing an HTTP request: libcurl code 6 meaning 'Couldn't resolve host name', error details: Couldn't resolve host 'metadata'"."."

### ISSUE THREAD

**Issue by KrisLee512** [google-research/vision_transformer#42]
this is the detailed error:
(VIT) llb@raypc:~/codes/vision_transformer$ python3 -m vit_jax.train --name ViT-B_16-cifar10_`date +%F_%H%M%S` --model ViT-B_16 --logdir /tmp/vit_logs --dataset cifar10
2020-12-05 09:46:07,527 [INFO] vit_jax.log: Namespace(accum_steps=8, base_lr=0.03, batch=512, batch_eval=512, copy_to=None, dataset='cifar10', decay_type='cosine', eval_every=100, grad_norm_clip=1, logdir='/tmp/vit_logs', mixup_alpha=0, model='ViT-B_16', name='ViT-B_16-cifar10_2020-12-05_094606', optim_dtype='bfloat16', output=None, prefetch=2, progress_every=10, shuffle_buffer=200000, tfds_data_dir=None, tfds_manual_dir=None, total_steps=None, vit_pretrained_dir='.', warmup_steps=500)
2020-12-05 09:46:08,183 [INFO] vit_jax.log: Available devices: [GpuDevice(id=0), GpuDevice(id=1), GpuDevice(id=2), GpuDevice(id=3)]
2020-12-05 09:46:08.185488: W tensorflow/core/platform/cloud/google_auth_provider.cc:184] All attempts to get a Google authentication bearer token failed, returning an empty token. Retrieving token from files failed with "Not found: Could not locate the credentials file.". Retrieving token from GCE failed with "Failed precondition: Error executing an HTTP request: libcurl code 6 meaning 'Couldn't resolve host name', error details: Couldn't resolve host 'metadata'".
how to solve this problem,thanks!!!!

**Comment by andsteing** [comment]
Maybe you run into problems connecting to a server for downloading the `cifar10` dataset? If that is the reason the the command `python -c 'import tensorflow_datasets as tfds; tfds.builder("cifar10").download_and_prepare()'` should result in the same error message. Make sure to have internet connection from the computer where you're running the code, or prepare the dataset on another computer and then copy the `~/tensorflow_datasets` directory.

**Comment by lyw615** [comment]
> Maybe you run into problems connecting to a server for downloading the `cifar10` dataset? If that is the reason the the command `python -c 'import tensorflow_datasets as tfds; tfds.builder("cifar10").download_and_prepare()'` should result in the same error message. Make sure to have internet connection from the computer where you're running the code, or prepare the dataset on another computer and then copy the `~/tensorflow_datasets` directory.

I have prepared cifar10 in local, but have no idea to prevent the program  connecting  tensorflow_datasets.  Does any args tell the program getting data from local?  Or any operation to function get_data of  input_pipeline.py  can achieve it ?

**Comment by Chenyu-Tian** [comment]
I met exactly the same problem, and I found it is a mistake of version. I replace the version to `tensorflow-datasets=2.1.0` and `tensorflow=2.3.0`. Problem solved!
